### DISCUSSION TITLE
Getstream

### DISCUSSION THREAD

**Discussion by Abdel-ilah** [skydoves/chatgpt-android#60] [General]
Hello,
thank you for great work

just a question please,
When the trial period of getstream.io ends, does the application continue to work?

Thank you

**Comment by skydoves** [comment]
Sorry for the late response. When the trial period of getstream.io ends, you should try to using a paid, or you can apply to the [Maker Account](https://getstream.io/maker-account/), which supports free capabilities for small team.
